install.xml

```diff
@@ -34,7 +35,7 @@
 	</phrases>
 
 	<blocks>
-		<block title="Our clients" name="clients_on_map" position="bottom" type="smarty"  classname="box-clear" filename="block.clients_on_map"><![CDATA[]]></block>
+		<block title="Our clients" name="clients_on_map" position="bottom" type="smarty"  classname="box-clear" filename="block.clients_on_map"><[CDATA[]]></block>
```

Comment:
Not sure if it is right to remove `!` in CDATA.. Any comments why is that?
Thanks.